### ISSUE TITLE
A11y: Shipping Info page loses content when zoomed in

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5256]
On the Shipping Info page of the checkout, when you zoom greater that 125%, the Summary and cart on the right hand side disappears. This is a failure of SC 1.4.10: [https://www.w3.org/WAI/WCAG22/Understanding/reflow.html|https://www.w3.org/WAI/WCAG22/Understanding/reflow.html|smart-link]  All content displayed, should still be visible when zoomed up to 400%.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39231**



[created from NEXT-39231, comment 506350]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from NEXT-39231, comment 511132]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-39231, comment 516501]
